contrib/segyconverter/segy_sample_files.ipynb

```diff
@@ -0,0 +1,461 @@
KLAVuLJr2ggsqapHkrwc+EqSY6vqiXn6LgeWAyxZsmRcJUuSJI3N2I+gJXkP8EbgnVVVAFW1paoe6ZZXA/cDR8/Xv6pWVNVsVc3OzMyMqWpJkqTxGWtAS3Iy8EHgTVX1y6H2mSQLuuUjgKOAB8ZZmyRJUiv6/JqNq4DvAi9MsiHJGcBngQOAlXO+TuM1wF1J1gDXAmdW1aPzvvGY7Lfvgt1qt8/e2UeSpEno7Rq0qjptnuZLdrDvdcB1fdXym1j7ibnfEGKfaewjSdIk+CQBSZKkxhjQJEmSGmNAkyRJaowBTZIkqTEGNEmSpMYY0CRJkhpjQJMkSWqMAU2SJKkxBjRJkqTGGNAkSZIaY0CTJElqjAFNkiSpMQY0SZKkxhjQJEmSGmNAkyRJaowBTZIkqTEGNEmSpMYY0CRJkhpjQJMkSWqMAU2SJKkxBjRJkqTGGNAkSZIaY0CTJElqjAFNkiSpMQY0SZKkxhjQJEmSGmNAkyRJaowBTZIkqTG9BrQklybZlOTuobbnJ1mZ5N7u50Fde5J8Jsl9Se5K8rI+a5MkSWpV30fQLgNOntP2YeBbVXUU8K1uHeD1wFHdazlwUc+1SZIkNanXgFZVtwCPzmk+Bbi8W74c+MOh9itq4HvAgUkO7bM+SZKkFk3iGrRDqmpjt/xT4JBueRHwk6H9NnRtkiRJU2WiNwlUVQG1O32SLE+yKsmqzZs391SZJEnS5EwioD28/dRl93NT1/4QcNjQfou7tl9TVSuqaraqZmdmZnovVpIkadwmEdBuAE7vlk8HvjrU/kfd3ZzHA48PnQqVJEmaGn1/zcZVwHeBFybZkOQM4JPA65LcC7y2Wwe4EXgAuA+4GPjjPmuTRrHfvgt2q90+9rGPfeyzd/cZlwwuA9szzc7O1qpVqyZdhiRJ0i4lWV1Vs6Ps65MEJEmSGmNAkyRJaowBTZIkqTEGNEmSpMYY0CRJkhpjQJMkSWqMAU2SJKkxBjRJkqTGGNAkSZIaY0CTJElqzB79qKckm4Efj+FXHQz8zRh+T8scA8cAHANwDMAxAMcAHAPY/TH4vaqaGWXHPTqgjUuSVaM+O2tv5Rg4BuAYgGMAjgE4BuAYQL9j4ClOSZKkxhjQJEmSGmNAG82KSRfQAMfAMQDHABwDcAzAMQDHAHocA69BkyRJaoxH0CRJkhpjQJsjyaVJNiW5e6jt+UlWJrm3+3nQJGvs2w7G4ONJHkqypnu9YZI19inJYUluSrIuydokZ3XtUzMPdjIG0zQPnpvk9iTf78bgE1374UluS3JfkquT/M6ka+3LTsbgsiQ/GpoHyyZda9+SLEhyZ5KvdetTMw+2m2cMpmoeJHkwyQ+6v3VV19bb54IB7ZkuA06e0/Zh4FtVdRTwrW59b3YZzxwDgAuraln3unHMNY3TVuCcqjoGOB54f5JjmK55sKMxgOmZB1uAk6rqpcAy4OQkxwN/zmAMfh/4GXDGBGvs247GAODfD82DNZMrcWzOAtYPrU/TPNhu7hjA9M2Df9r9rdu/WqO3zwUD2hxVdQvw6JzmU4DLu+XLgT8ca1FjtoMxmBpVtbGq7uiWn2TwD9Iipmge7GQMpkYN/Lxb3ad7FXAScG3XvrfPgx2NwVRJshj4l8B/7dbDFM0DeOYY6O/09rlgQBvNIVW1sVv+KXDIJIuZoA8kuas7BbrXnt4blmQpcBxwG1M6D+aMAUzRPOhO6awBNgErgfuBx6pqa7fLBvby4Dp3DKpq+zw4r5sHFybZd4IljsN/Bj4IPNWtv4Apmwc8cwy2m6Z5UMA3k6xOsrxr6+1zwYC2m2pw2+vU/R8kcBFwJIPTHBuB8ydbTv+S7A9cB5xdVU8Mb5uWeTDPGEzVPKiqbVW1DFgMvAJ40YRLGru5Y5DkxcC5DMbiHwHPBz40wRJ7leSNwKaqWj3pWiZlJ2MwNfOg86qqehnwegaXfbxmeOOz/blgQBvNw0kOBeh+bppwPWNXVQ93/1A/BVzM4MNqr5VkHwbB5Mqqur5rnqp5MN8YTNs82K6qHgNuAl4JHJhkYbdpMfDQxAobo6ExOLk7BV5VtQX4b+zd8+AE4E1JHgS+yODU5qeZrnnwjDFI8t+nbB5QVQ91PzcBX2bw9/b2uWBAG80NwOnd8unAVydYy0Rsn4CdNwN372jfPV13fcklwPqqumBo09TMgx2NwZTNg5kkB3bLzwNex+BavJuAt3a77e3zYL4xuGfoAykMrrnZa+dBVZ1bVYurailwKvDtqnonUzQPdjAG75qmeZBkvyQHbF8G/jmDv7e3z4WFu95luiS5CjgRODjJBuBjwCeBa5KcAfwYePvkKuzfDsbgxO4W6gIeBN43sQL7dwLwbuAH3bU3AB9huubBjsbgtCmaB4cClydZwOB/Zq+pqq8lWQd8McmfAXcyCLJ7qx2NwbeTzAAB1gBnTrLICfkQ0zMPduTKKZoHhwBfHmRRFgL/o6q+keT/0tPngk8SkCRJaoynOCVJkhpjQJMkSWqMAU2SJKkxBjRJkqTGGNAkSZIaY0CTtNdL8n9G2Oc7SWa75Ru3f/+XJE2C34Mmaa9XVX+wm/u/oa9aJGkUHkGTtNdL8vPu54ndkbJrk9yT5MruW9Dn7v9gkoOTLE2yPsnFSdYm+Wb3jfokOTLJN7oHJ/+vJFP3nE5J/TGgSZo2xwFnA8cARzB4asLOHAX8RVUdCzwG/KuufQXwb6vq5cC/Az7XT7mSppGnOCVNm9uragNA9xirpcCtO9n/R1W1/XFXq4GlSfYH/gD40tABuH37KVfSNDKgSZo2W4aWt7Hrfwfn7v88BmcfHquqZc9ybZIEeIpTknZbVT0B/CjJ2wAy8NIJlyVpL2JAk6TfzDuBM5J8H1gLnDLheiTtRVJVk65BkiRJQzyCJkmS1BgDmiRJUmMMaJIkSY0xoEmSJDXGgCZJktQYA5okSVJjDGiSJEmNMaBJkiQ15v8DYqF0avnwe1gAAAAASUVORK5CYII=\n",
```

Comment:
I'll review this later again - dep on image size we may want to host on Azure.

Reply:
This is the output from the scatter plot of the segy files so I don't think there is a way to host on azure storage. Do you want the notebooks checked in with output data? or should the user be expected  to run the notebook to see the output?

Reply:
The notebook with these images is only 60K so I think it shouldn't add too much bloat